packages/jest-environment-puppeteer/tests/__fixtures__/invalidProduct.js

```diff
@@ -1,5 +1,5 @@
 module.exports = {
   launch: {
-    product: "foobar",
```

Comment:
@gregberge I assume the file should have been renamed to `invalidBrowser.js`? Maybe there are other instances as well.

Reply:
Yeah, it's just a name.